build.gradle.kts

```diff
@@ -19,15 +19,27 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.springframework.boot:spring-boot-starter-validation")
     implementation("org.springframework.boot:spring-boot-starter-actuator")
+    implementation("junit:junit:4.13.1")
+
+    compileOnly("org.projectlombok:lombok")
+    annotationProcessor("org.projectlombok:lombok")
 
     implementation("org.springdoc:springdoc-openapi-data-rest:1.7.0")
     implementation("org.springdoc:springdoc-openapi-ui:1.7.0")
 
     testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation("org.pitest:pitest:1.4.10")
```

Comment:
we're not using pitest, so should we be adding it here?

Reply:
this is for future use and also before I push I run build pietist. At the moment they fail but tell me how many mutations were tested

Reply:
That's not good - if we've got failing mutation tests out of the box, it would be good not to include it but to ticket it up as a thing to do